workflow/pred_temperature_LSTM.ipynb

```diff
@@ -22,6 +22,16 @@
     "- Evaludate model"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # check this doc if not https://era5cli.readthedocs.io/en/latest/instructions.html#register-at-copernicus-climate-data-service
</pre><p>This link is broken. The right link is now https://era5cli.readthedocs.io/en/stable/getting_started/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Thanks! I will update it!</p>